ci/scripts/github/build_wheel.sh

```
@@ -37,12 +37,12 @@ for NAT_PACKAGE in "${NAT_PACKAGES[@]}"; do
 done
 
 # Build all examples with a pyproject.toml in the first directory below examples
-for NAT_EXAMPLE in ${NAT_EXAMPLES[@]}; do
+for NAT_EXAMPLE in "${NAT_EXAMPLES[@]}"; do
```

Shellcheck (0.11.0)

[info] Possible misspelling: NAT_EXAMPLES may not be assigned. Did you mean NAT_EXAMPLE?

(SC2153)